tests/unit/ext/auth/database.py

```diff
@@ -15,6 +13,18 @@
 
 
 @pytest.fixture
+def dbauth():
+    from haas.ext.auth import database
```

Comment:
Nevermind, answered my own question. The scope of this is to have a 'clean' room for extensions.